geomstats/learning/frechet_mean.py

```diff
@@ -333,6 +333,50 @@ def _adaptive_gradient_descent(
 
     return current_mean
 
+def _circle_mean (points):
+    """Determine the mean on a circle.
+    
+    Data are expected in radians in the range [-pi, pi). The mean is returned
```

Comment:
You could provide this information by adding a `Parameters` section in the docstring, see here for example:
https://github.com/geomstats/geomstats/blob/048a99dc9dff3f86e42025264ac0811ed6888f7c/geomstats/geometry/lie_algebra.py#L55